.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description: Request a missing feature
+title: '[Feature] '
+labels: ['feature', 'needs triage']
+body:
+  - type: markdown
+    id: intro-md
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request! Please ensure you provide as much information as possible so we can better understand what you want us to add and so we can come up with the best solution for everyone.
```

Comment:
Maybe change this to "what your posing to be added and so we can come up with the best solution for everyone." so it takes away the expectation that what is asked will get deployed